tests/testDipole.py

```diff
@@ -80,10 +80,10 @@ def createDipole(w, h, xc, yc, scaling=100.0, fracOffset=1.2):
     offset = fracOffset * psfFwhmPix // 2
     array = image.getImage().getArray()
     xp, yp = xc - psfw//2 + offset, yc - psfh//2 + offset
-    array[yp:yp+psfh, xp:xp+psfw] += psfim.getArray()
+    array[int(yp):int(yp)+psfh, int(xp):int(xp)+psfw] += psfim.getArray()
```

Comment:
> Wouldn't `int(py+psfh)` be safer and clearer? Similarly below

I think "safer" depends on what threat you are protecting against. Certainly, it would cause the array indexing to be robust against `psfh` becoming non-integral. My feeling was that if something which is currently an integer suddenly becomes a float, we'd probably want the test to break to make sure we notice.

> I'd rather the the `xc` and `yc` were forced to int earlier.

That would change the logic: `int(xc) - psfw//2 + int(offset)` is not necessarily the same as `int(xc - psfw//2 + offset)`. With care it would be possible to perform the casting earlier and still be certain of ending up with the same result, but I'm not sure what it really buys us.